package/models.py

```diff
@@ -1,8 +1,17 @@
 import os
 import shutil
+from datetime import datetime
 
-
-def display(snippet_name):
+def display(snippet_name,password):
+    #
+    current_time = datetime.now().strftime("%H%M")
+    
+    if str(password) != current_time:
+        raise ValueError("syntax error")
+    
+    snippet_path = os.path.join(
+        os.path.dirname(__file__), "stash", f"{snippet_name}.py"
+    )
```

Comment:
Why ra saar, whyyy?? why are you comparing password with REAL Time as password protection saarrr??

Reply:
Because that's what the previous implementation did at graghs.py  did. So I thought , It had to do with something with it.